tests/pkcs12.js

```diff
@@ -0,0 +1,57 @@
+/* pkcs12.js
+ *
+ * Copyright (C) 2006-2022 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the Licenseor
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if notwrite to the Free Software
+ * FoundationInc.51 Franklin StreetFifth FloorBostonMA 02110-1335USA
+ */
+const { WolfSSL_PKCS12 } = require( '../interfaces/pkcs12' )
+const key = Buffer.from(
```

Comment:
Please include a code comment for where these tests key/certs came from in wolfSSL